src/js/page/ui/floating-action-button.js

```diff
@@ -1,30 +1,25 @@
-import { strToEl } from '../utils';
-import Ripple from './ripple';
+import { strToEl } from '../utils.js';
+import Ripple from './ripple.js';
 
 export default class FloatingActionButton {
   constructor({ title, href, iconSvg, major = false }) {
+    // prettier-ignore
     this.container = strToEl(
       (href ? '<a>' : '<div role="button" tabindex="0">') +
         iconSvg +
-      (href ? '</a>' : '</div>') +
-    '');
+      (href ? '</a>' : '</div>')
+    );
 
     const classes = ['floating-action-button'];
 
-    if (href) {
-      this.container.href = href;
-    }
-    if (title) {
-      this.container.setAttribute('title', title);
-    }
-    if (major) {
-      classes.push('major-floating-action-button');
-    }
+    if (href) this.container.href = href;
```

Comment:
I used to be against single line ifs, but not anymore. This is so much easier to read 😄